frontend/src/static/js/main.js

```diff
@@ -245,8 +254,8 @@ window.app = new window.Vue({
     if (!this.isTabActive) {
       window.removeHash('tabAuthor');
       window.removeHash('tabRepo');
-      window.addHash('tabOpen', this.isTabActive);
-      window.encodeHash();
     }
+    window.addHash('tabOpen', this.isTabActive);
```

Comment:
Why this change?

Reply:
To always have a `tabOpen` hash in the URL, whether be it `true` or `false`. 
Previously there will only be a `tabOpen` hash when it is `false`.

Reply:
OK, reasonable.